### PR TITLE
Added: Delete agreement details

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: vipps-recurring-api-faq.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
-- [Can the charge for an agreement be changed?](#can-the-charge-for-an-agreement-be-changed-)
-- [Are there any limits on charging a user?](#are-there-any-limits-on-charging-a-user-)
-- [When can I send charges for a user?](#when-can-i-send-charges-for-a-user-)
-- [Can a user cancel the agreement through the Vipps app](#can-a-user-cancel-the-agreement-through-the-vipps-app)
-- [How does a user see any charges I send?](#how-does-a-user-see-any-charges-i-send-)
-- [If a user changes the default payment card in Vipps, can new charges be made to that card?](#if-a-user-changes-the-default-payment-card-in-vipps--can-new-charges-be-made-to-that-card-)
-- [If a user's card expires: What happens on the next charge?](#if-a-user-s-card-expires--what-happens-on-the-next-charge-)
-- [Settlement](#settlement)
-- [Invoicing](#invoicing)
+* [At what time during the day are charges made](#at-what-time-during-the-day-are-charges-made)
```
Comment (tht13) [inline vipps-recurring-api-faq.md]: can you please stop overwriting this every time?

File: vipps-recurring-api-faq.md [hunk 2]
```diff
@@ -63,6 +65,14 @@ the user.
 No, the user needs to contact the Merchant which can then cancel or modify the
 agreement as they see fit.
 
+## What happens to charges if the user cancels the agreement?
+All charges in a `PENDING` or `DUE` state will be cancelled if the agreement is stopped.
+
+## What happens to pending charges if the user deletes the payment card?
+We are going to update the card handling logic on our side and won't let a user
```
Comment (tht13) [inline vipps-recurring-api-faq.md]: please remove this, we don't need to lay out our roadmap
you can write:
`The user is responsible for keeping their payment sources up-to-date. They will receive a push notification if a payment fails which informs them to update their payment source. We are looking to improve user experience here in the future`

Comment (cloveras) [reply vipps-recurring-api-faq.md]: An open roadmap can be useful, and this has already been communicated externally - but I can of course remove it.

Comment (cloveras) [reply vipps-recurring-api-faq.md]: An open roadmap can be useful, and this has already been communicated externally - but I can of course remove it.

File: vipps-recurring-api-faq.md (hunk 1 above)
Comment (cloveras) [reply vipps-recurring-api-faq.md]: I will make a feeble attempt.

**Pull request merged**